src/games/strategy/triplea/ResourceLoader.java

```diff
@@ -184,7 +184,7 @@ public URL getResource(final String inputPath) {
     String path = resourceLocationTracker.getMapPrefix() + inputPath;
     return getMatchingResources(path).stream().findFirst().orElse(
         getMatchingResources(inputPath).stream().findFirst().orElseGet(
-            ()->null));
+            () -> null));
```

Comment:
Small annoyance - there are a number of unrelated changes in this PR. Particularly for bug fixes, but also in general, please put unrelated changes on their own branches.